//存放所有页面英文内容，各页面加载对应页面的即可
var English = {
  
  'index_index': {
    'changeLanguage': 'Language/语言', 'iProteomePlatform': 'iProteome Platform', 'Login': 'Login', 'AboutUs': 'About us', 'Staff': 'Staff', 'customer':'Customer','welcome': 'Welcome' , 'click_to_register': 'Please click on your avatar to register', 'click_to_login': 'Please click on your avatar to login' ,
    'unit1': 'Human Phenome Institute Fudan University', 'unit2': 'School of Life Sciences Fudan University', 'unit3': 'State Key Laboratory of Genetic Engineering Fudan University', 'Loading': 'Loading', 'Jumping': 'Jumping', 'Warning': 'Warning', 'Warning_not_registered': 'You are not registered, please log in after registration! Going to register now?', 'Warning_not_staff': 'Sorry, you are not our staff!', 'Warning_not_login': 'Please log in first', 'Cancel': 'Cancel', 'Confirm': 'Confirm', 'warn': 'Warning', 'warn_content': 'Request timed out, please try again later!', 'confirmText': 'Confirm', 'cancelText': 'Cancel'
  },
  'srvPurchase_srvPurchase': {
    'addProject': 'Add Project', 'backToHomepage': 'Back to Home', 'basicInfo': 'Basic Information', 'basicInfoContent': 'iProteome Platform is a proteomics research platform formed by the the Human Phenome Institute of Fudan University.The Human Phenome Institute (HuPI) was founded in 2017 as an independent research organization in  Fudan University. The HuPI leads the International Human Phenome Project (Phase 1) with support from the first Shanghai Municipal Science and Technology Special Funds for Major Projects, serves the significant demands of launching large international science projects by China and building a major world center for science and innovation in Shanghai, and is one of the key pillars of “One Plan and Two Centers” top-level design by the Zhangjiang-Fudan International Innovation Center.As an important measure for Shanghai to build a globally influential centerof science and technology innovation, the project will bring together more than 100 scientists at home and abroad to conduct cross-scale and multi-dimensional research on the phenotypic characteristics of the human body from macro to micro. The establishment of the first research platform for human phenome formulate a standardized technical system comprehensively for human health phenotypes in China, and construct the first Chinese population phenome map and database.The HuPI currently has 26 investigators: one Academician of Chinese Academy of Sciences, one Distinguished Professor in “Cheung Kong Scholars”, seven Distinguished Young Scholars of National Natural Science Funds，three awardees of “Thousand Talents Plan” and five awardees of “Thousand Young Talents Plan”. The iProteome proteomic analysis platform has more than 50 employees, with key members in the fields of biology and medical doctors and masters. The team has been working in the field of proteomics for many years, and is committed to the development of new techniques and new methods of proteomics and transcriptional regulation research based on mass spectrometry/proteomics technology. In the development and application of large-scale, low-abundance, high-precision proteomics technology. It has established the most efficient proteomics coverage platform in the world, and invented a large-scale identification of endogenous transcription factors, activity quantification methods and single transcription factor transcriptional regulation complex mass spectrometry research mode. The iProteome team masters the core technology of proteomics and builds a complete solution for sample preparation, mass spectrometry and data analysis based on user research and medical needs.', 'samPre_MS': 'Sample Preparation/MS', 'samPre_MSContent': 'We have established the most efficient proteomic platform in the world, which can identify more than 8,000 proteins within 3 hours. Based on this, we have mapped the landscape of diffuse-type gastric cancer proteome,  which provides valuable material for understanding the molecular mechanism of gastric cancer and provides a basis for accurate treatment of gastric cancer patients. We have also invented a method for large-scale identification of endogenous transcription factor complexes, and used this method to map the mouse tissue transcription factor atlas，and revealed the principle of circadian clock in mouse liver.', 'dataAnalysis': 'Data Analysis', 'dataAnalysisContent': 'Firmiana, a one-stop proteomic data processing and integrated omics analysis cloud platform that allows scientists to deposit mass spectrometry (MS) raw files, perform proteome identification and quantification online, carry out bioinformatics analyses, extract knowledge, and visualize results using a biologist friendly web interface without the need for programming expertise. Currently, there are more than 20 countries and 300 laboratories use our Firmiana platform, covering a total of 30,000 raw data. Meanwhile, we have established the human proteome sample Internet of Things. We have implemented automated management from sample collection to sample processing, sample testing, data processing, and diagnostic reporting.', 'standard': 'Proteome Industry Standard', 'standardContent': 'The iProteome Platform has led to develop data standards and established the national standards for proteomics in China. At the same time, it is the only designated international proteomics technology and material standards development team for the MicroArray Quality Control (MAQC) which led by the FDA, NIST, CFDA, and Metrology Institute, and master the international discourse rights of the proteome industry.', 'teamIntro': 'Team Introduction', 'teamIntroContent': 'The iProteome team undertakes many research projects supported by the National Natural Science Foundation of China (NSFC), and participates in several international and domestic major science programs, such as Chinese Human Proteome Project (CNHPP), the International Massive Analysis and Quality Control Society (MAQC), etc. We have accumulated considerable experience of research and clinical trial, and possessed strong transformation capability of scientific and technological achievements. As the frontrunner in the field of proteomics, our team was  involved in the standard establishment of proteomics. With absolute technical advantages, we masters the right to speak in the industry. In the past 2 years,  our team has published 16 papers as correspondence authors or first authors in high-level journals, such as Nature Biotechnology, Cell, Journal of Experimental Medicine, Molecular Cell, Nature Communications, PNAS, Molecular & Cellular Proteomics, etc. The iProteome platform undertakes the tasks of cultivating specialized talents for Fudan University and Shanghai teaching and research, and provides scientific research and technical cooperation for the society. The iProteome platform is dedicated to providing qualitative and quantitative analysis, modified proteomic analysis and transcription factor DNA binding activity spectrum analysis, which have the ability of one-stop proteomic analysis including sample preparation, proteomics detection, data analysis. With advanced experimental hardware facilities and professional technical support，the iProteome platform of Fudan University has established a diversified and stable research cooperation group including biotechnology companies, universities and research institutes.', 'instrument': "Platform's Instruments", 'instrumentContent':'The current platform includes five Q Exactive HF-X Hybrid Quadrupole-Orbitrap Mass Spectrometers and two Orbitrap Fusion Lumos Tribrid Mass Spectrometer.','tabBar': { 'srvIntro': 'Service Intro', 'intelligentInquiry': 'Inquiry', 'ServicePurchase': 'Purchase', 'me': 'Me' },
  },
  'basicInfo_basicInfo': {
    'sampleInfo': 'Sample Information', 'projectName': 'Project Name', 'sampleType': 'Sample Type', 'bioClassification': 'Biological classification', 'Taxon': 'Taxon', 'TaxonID': 'TaxonID', 'strainName': 'Strain Name', 'system': 'System', 'organ': 'Organ', 'status': 'Status', 'cellType': 'Cell Type', 'cell': 'Cell', 'fluid': 'Fluid', 'sampleNumber': 'Number of Samples', 'next': 'Next', 'sourceType': [
      { data: 'Tissue', checked: false },
      { data: 'Cell', checked: false },
      { data: 'Fluid', checked: false },
    ], 'Loading': 'Loading', 'Warn': {
      'projectName': 'Please fill in the project name', 'sourceType': 'Please fill in the sample type', 'Species': 'Please select a taxonomy', 'TaxonName': 'Please select a species', 'TaxonID': 'Please select TaxonID', 'strainName': 'Please select Strain Name', 'System': 'Please select a system', 'Organ': 'Please select an organ', 'Status': 'Please select a status', 'CellType': 'Please select the cell type', 'cellName': 'Please select a cell', 'FluidName': 'Please choose body fluid', 'totalConditionNumber': 'Please fill in the number of samples'
    }, 'Warning': 'Warning', 'Cancel': 'Cancel', 'Confirm':'Confirm',
    
  },
  'condition_condition':{
    'sample': 'Sample', 'age_growthCycle': 'Age/Growth cycle', 'gender': 'Gender', 'targetGeneNumber': 'Total number of target genes', 'targetGene': 'Target Gene', 'TaxonID': 'TaxonID', 'genotype': 'Genotype', 'sampleStatus': 'Sample status', 'storageForm': 'Storage form', 'amount': 'Amount', 'treatment': 'Treatment', 'dose': 'Dose', 'duration': 'Duration', 'fractionNumber': 'Fraction Number', 'bioRepet': 'Biological repetition', 'techRepet': 'Technical repetition', 'previous': 'Previous', 'saveAndNext': 'Save and Next', 'AgeUnit': ['day', 'week', 'mounth', 'year'], 'Gender': ['NA', 'Male', 'Female', 'Androgynous', 'Asexual'], 'SampleStatus': ['Peptide', 'Protein', 'Cell', 'Tissue', 'Paraffin section(FFPE)', 'Frozen slice', 'Body fluid'], 'StorageForm': ['Dry Powder', 'Solution ABC', 'Solution RIPA', 'Solution NETN', 'Solution 8MUREA', 'Solution PBS','Solution Other','Solid Cell','Solid Tissue','Solid Slice','Fluid Solution'],'warn': 'Warning', 'Warn': {
      'ageNum': 'Please fill in the age/growth cycle!', 'ageUnit': 'Please select age/growth cycle unit!', 'gender': 'Please select gender!', 'genenum': 'Please select the target gene number!', 'sampleStatus': 'Please select the sample status!', 'storeForm': 'Please select a storage form!', 'amount': 'Please fill in the sample size!', 'amountUnit': 'Please select the sample unit!', 'sampleConcentration': 'Please fill in the sample concentration!', 'concentrationUnit': 'Please select the sample concentration unit!', 'sampleVolume': 'Please fill in the sample volume!', 'volumeUnit': 'Please select the volume unit!', 'treatmentMethods': 'Please select a stimulus condition!', 'treatmentDetail': 'Please choose the stimulation method!', 'paramtype': 'Please choose a dose!', 'paramAmount': 'Please fill in the number of doses!', 'paramUnit': 'Please select the dosage unit!', 'durationTime': 'Please fill in the duration!', 'durationUnit': 'Please select the duration unit!', 'fractionNumber': 'Please fill in the fraction number!', 'technicalRepetition': 'Please fill in the technical repetition number!'
    }, 'Cancel': 'Cancel', 'Confirm': 'Confirm', 'placeholder_dryPowder': 'total amount(Dry Powder)', 'placeholder_number': 'number', 'placeholder_weight': 'weight', 'placeholder_volume': 'volume', 'showLoading': 'Loading'
  },
  'serviceType_serviceType': {
    'serviceType': 'Service Type','navigationBarTitle':'Services',
    'serviceTypeList':[
    {
      title: 'Qualitative Proteomics',id: 1, subtitle: [{ name: 'Proteome profiling', checked: false }, { name: 'Protein analysis after pull-down(IP,Co-IP)', checked: false }]
    },
    {
      title: 'Quantitative Proteomics ', id: 2, subtitle: [{ name: 'Label-free quantification', checked: false }, { name: 'iTRAQ labelling quantification', checked: false }, { name: 'TMT labelling quantification', checked: false }, { name: 'SILAC labelling quantification', checked: false }]
    },
    {
      title: 'PTM Analysis', id: 3, subtitle: [{ name:'Qualitative analysis of protein modification sites',checked:false},{ name: 'Phosphorylation Proteomics analysis', checked: false }, { name: 'Ubiquitylation Proteomics analysis', checked: false }, { name: 'Glycosylation Proteomics analysis', checked: false }, { name: 'Acetylation  Proteomics analysis', checked: false }]
    },
    {
      title: 'Transcription factor - DNA binding activity analysis', id: 4, 'subtitle': [{ name: 'Transcription factor -DNA binding activity analysis', checked: false }]
    }
    ], 'next': 'Next', 'warn': 'Warning', 'warn_content': 'Please select at least one service！','Cancel': 'Cancel', 'Confirm': 'Confirm',
  },
  'order_sampleInfo': {
    'sampleInfo': 'Sample Information', 'analysisTime': 'Analysis Time', 'AnalysisTime': ['30min', '75min', '150min'], 'dataAnalysis': 'Data Analysis', 'DataAnalysis': ['No', 'Qualitative/quantitative analysis', 'Differential protein analysis', 'Path analysis','High-end custom'],'projectNotes': 'Project Notes', 'address': 'Address', 'contactNumber': 'Contact Number', 'submitOrder': 'Submit Order', 'note': 'Please fill in carefully!', 'warn':'warning',
    'Warn': {
      'analysisTime': 'Please select the instrument analysis time!', 'dataAnalysis': 'Please select data analysis!', 'address': 'Please fill in the barcode mailing address!', 'contactNumber': 'Please fill in the contact number!'
    }, 'note': 'Please fill in carefully!', 'Cancel': 'Cancel', 'Confirm': 'Confirm', 'partyA': 'Commissioning Party(Party A)', 'legalRepresentative': 'Legal Representative', 'projectContactPerson': 'Project Contact Person', 'postcode': 'Postcode', 'fax': 'Fax',
    },
  'order_orderInfo': {
    'MoneyList': [
      { category: "Sample Preparation", content: [], unitPrice: [], amount: '', money: '' },
      { category: " Mass Spectrometry", content: '', unitPrice: '', amount: [], money: '' },
      { category: "Data Analysis", content: '', unitPrice: '', amount: '', money: '' },
    ],
    'orderInfo': 'Order Information', 'sampleInfo': 'Sample Information', 'projectName': 'Project Name', 'sourceType': 'Sample Type', 'species': 'Species', 'Taxon': 'Taxon', 'TaxonID': 'TaxonID', 'strainName': 'Strain Name', 'system': 'System', 'organ': 'Organ', 'status': 'Status', 'cellType': 'Cell Type', 'cellName': 'Cell', 'fluidName': 'Fluid', 'sample': 'Sample', 'age': 'Age', 'gender': 'Gender', 'targetGeneNumber': 'Target Gene Number', 'targetGene': 'Target Gene', 'geneID': 'GeneID', 'geneSymbol': 'GeneSymbol', 'taxonID': 'TaxonID', 'Genotype': 'Genotype', 'sampleStatus': 'Sample Status', 'storeForm': 'Storage Form', 'amount': 'Sample Amount', 'concentration': 'Concentration', 'volume': 'Volume', 'treatment': 'Treatment', 'dose': 'Dose', 'duration': 'Duration', 'fractionNumber': 'Fraction Number', 'bioRepet': 'Biological Repetition', 'techRepet': 'Technical Repetition', 'selectedServices': 'Selected Services', 'analysisTime': 'Analysis Time', 'dataAnalysis': 'Data Analysis', 'address': 'Address', 'serviceType': 'Service Type', 'serviceContent': 'Service Content', 'number': 'Number', 'unitPrice': 'Unit Price', 'Amount': 'Amount(yuan)', 'totalTechRepet': 'Total Technical repetition', 'serviceNum': 'Service Number', 'sampleNum': 'Sample Number', 'techRepet': 'Technical Repetition Number', 'totalAmount': 'Total Amount', 'discount': 'Discount', 'confirm': 'Confirm', 
  },
  'contract_contractIndex': {
    'navigationBarTitle': 'Contract Preview', 'title': 'Technical service contract', 'version': '(2018 version)', 'include': '(Including technical training and intermediary)', 'projectName': 'Project Name', 'projectNumber':'Project Number','PartyA': 'Party A', 'PartyA_content': 'Fudan Universiy', 'PartyB': 'Party B', 'PartyB_content': 'Fudan Universiy Human Phenome Institute', 'signingPlace': 'Place of signing', 'signingPlace_content':'Yangpu District, Shanghai','signingDate': 'Date of signing', 'to':'to','validityPeriod': 'Validity Period', 'unit1': 'Shanghai Municipal Science and Technology Commission', 'unit2': 'Shanghai Municipal Administration for Industry and Commerce','next':'Next'
  },
  'contract_fillingExplanation': {
    'navigationBarTitle': 'Contract Preview','title': 'Explanatory notes\n(Stamp Sticking Place)', 
    'fillingExplanationList':[
      '1.The method of filling contract registration number:\n \tThe contract registration number is filled by contract registration office.',
   '2.Technical service contract concluded by one party to solve the other’s technical problem with technical knowledge. The technical service contract include technical training contract and technical intermediary contract.\n \tTechnical training contract concluded one party entrust the other party to training some special project and professional training for their professional technical personnel.\n \tTechnical intermediary contract is provided by one party with knowledge, technique, experiment and information to the other party and a third party about their contact, introduction,organize industrial development.',
   '3.The planned research projects shall be filled out with the corresponding State Council ministries, provinces, autonomous regions, directly administered areas, or planned cities, prefectures, or municipalities (counties) at the project level, and the project plan that does not belong to the above shall be indicated as (/).',
     '4.The service content, modalities and requirements:\n \tThe characteristics of technical services, the scope of the target and the benefits; the difficulty of specific technical issues, the main technical and economic indicators; the specific practices, means, procedures and forms of delivery is included. It is a technical training contract and a technology intermediary contract. \n\tA technical training contract is a contract concluded by one of the parties to entrust the other party to the technical guidance and professional training of the designated professional and technical personnel for the specific project. \n\tA technology intermediary contract refers to a contract concluded by a party to establish a technical contract with a third party based on knowledge, technology, experience and information, to introduce, organize industrial development and provide services for the performance of the contract.',
     "5.Working conditions and Collaboration Matters:\n\tConsignor should offer some working conditions to assure consignee could complete the job perfectly including technical information and other necessary work conditions. \n\tIf it was a technic training agreement, consignor should offer promissory work yards, equipment and other conditions and manage them properly.",
    "6.Settlement of Disputes:\n\tIn the event of any dispute, controversy or claim (collectively, “dispute”) arising out of or relating to this Contract, or the breach, termination or invalidity hereof, the Parties shall attempt in the first instance to resolve such dispute through friendly consultations. According to Arbitration Law of the People's Republic of China, if the parties have concluded an arbitration agreement and one party institutes an action in a people's court, the people's court shall not accept the case, unless the arbitration agreement is null and void.",
     '7.Others:\n\tIf the contact was signed via intermediary agent, the intermediate contract must be attached to this contact as attachment. If both parties arrange the deposit, mortgage of property.',
   '8.The proxy must be issued while client entrust an agent to signed this contract.',
    '9.If there are any terms that the parties believe do not need to be filled out in this contact, “/” should be drawn in the corresponding blank space to indicate.',
    '10.This contact is in quadruplicate.'
    ],
    'next': 'Next'
  },
  'contract_contract':{
    'navigationBarTitle': 'Contract Preview', 'title': 'Technical Service Contract', 'contractNo': 'Contract No', 'partyA': 'Principal (Party A)', 'partyB': 'Commissioned Party(Party B)', 'partyB_content': 'Human Phenome Institute Fudan University', 'legalRepresentative_content': 'Chen Ding', 'projectContactPerson_content': 'Subei Tan', 'address_content': 'No.2005, Songhu Road, Yangpu District, Shanghai', 'telephone_content': '021-31240741', 'fax_content': '021-31240741', 'postcode_content': '200438', 'email': 'Email', 'email_content': 'iproteome@firmiana.org', 'legalOrPrincipal': 'Legal person/Principal agent', 'seal': 'Seal', 'sign': 'Sign', 'confirm': 'Confirm & Back', 'date': 'Date', 'jumping': 'Jumping', 'done': 'Done!', 'head1': "According to the 'Contract Law of the People's Republic of China', the contractual parties signed the contract for the technical service of the",'head2':'project.',
    'title1': '1.※Service content, methods and requirements', 
    'title1_description1': '(The technical training contract should fill in the training content and requirements, training plan, and progress. The technical intermediary contract should fill in the intermediary content and requirements.)\nParty A of this contract entrusts the',
    'title1_description2':"of Party B to carry out special technical services and pay the corresponding technical service remuneration.On the basis of fair and full expression of their respective wishes, the two sides reached the following agreement in accordance with the provisions of the Contract Law of the People's Republic of China, and they are jointly observed by both parties.",
    'Article1': 'Article 1',
    'Article1_description': 'Party A entrusts Party B with technical services as follows:',
    'Article1_content': '1.Sample preparation\n2.Mass spectrometry\n3.Data analysis',
    'Article2': 'Article 2',
    'Article2_description':'The parties determine: \nthe relevant nouns and technical terms involved in this contract and related annexes are defined and explained as follows:',
    'Article2_content':'No',
    'title2':'2.※Working conditions and cooperation matters',
    'Article3': 'Article 3',
    'Article3_description1':'In order to ensure the effective technical service work of Party B, Party A shall provide Party B with relevant materials, data, samples, batch numbers, etc., including:',
    'Article3_description2':'After receiving the samples provided by Party A, Party B shall test the quality of the samples, conduct mass spectrometry tests on the qualified samples, and communicate with Party A in time to confirm the sample delivery.',
    'Article3_sampleName':'Sample Name:',
    'Article3_sampleLotNumber':'Sample lot number:',
    'Article4': 'Article 4',
    'Article4_description':'The parties shall determine the confidentiality obligations to be observed in fulfilling this contract as follows:',
    'Article4_content':'1.Confidentiality: Test sample information, data, results, test prices and technical data.\n2.Confidentiality period: Party B is permanently confidential for Party A, and Party A is permanently confidential for Party B.\n3.Liability for disclosure: The loss is the responsibility of the leaking party.',
    'Article5': 'Article 5',
    'Article5_description':'Changes to this contract must be agreed by both parties and determined in writing. However, in one of the following circumstances, one party may submit a request for change of contractual rights and obligations to the other party, and the other party shall reply within 15 days; if it fails to reply within the time limit, it shall be deemed as:',
    'Article5_content':'1.No\n2.No\n3.No',
    'title3': '3.※Time limit, place and method of performance:',
    'title3_description1': 'This contract was performed from',
    'title3_description2': 'to',
    'title3_description3': 'at the Fudan University Proteome Platform (location).',
    'title3_description4': 'How this contract is fulfilled:',
    'Article6':'Article 6',
    'Article6_description1':'The two sides agreed,Party B shall complete all testing services by',
    'Article6_description2':", and Party B shall be responsible for providing Party A with a copy of the original test report issued by Fudan University's proteome platform.\nWithin 30 natural days after the contract is signed, Party A shall be responsible for providing qualified samples to Party B;\nParty B shall complete the mass spectrometry and data analysis of all samples within 360 natural days from the receipt of the qualified samples, and provide Party A with a copy of the original test report issued by the Fudan University Proteome Platform. If unexpected matters arise during the process, both parties will resolve the dispute through negotiation.",
    'Article7': 'Article 7',
    'Article7_description':'Both sides confirmed:',
    'Article7_content':'1.During the term of this contract, Party A shall use the new technical achievements completed by the technical service work submitted by Party B to be owned by both parties.\n2.During the term of this contract, Party B shall use the technical data and working conditions provided by Party A to complete the new technical achievements.',
    'Article8':'Article 8',
    'Article8_description1':'Both parties determine that during the term of this contract,Party A designates',
    'Article8_description2':'as the project contact of Party A,Party B designates',
    'Article8_description3':'as the project contact of Party B.The project contact assumes the following responsibilities:',
    'Article8_description4':'1.Sample transfer\n2.Information transfer',
    'Article8_description5':'nIf one party changes the project contact, it shall promptly notify the other party in writing. If it is not notified in time and affects the performance of this contract or causes losses, it shall bear corresponding responsibility.',
    'Article9':'Article 9',
    'Article9_description':'The parties have determined that the following circumstances have caused the performance of this contract to become unnecessary or impossible, and the contract can be terminated:',
    'Article9_content': '1.Force majeure;\n2.Party A no longer carries out this project;\n3.Party B no longer has the ability to execute this project.',
    'title4':'4.※Acceptance criteria and methods:',
    'title4_description1':'Technical service or technical training According to the protein platform testing standards, Party B will issue a project test report.',
    'title4_description2':'The warranty period for this contract service project is one year, and the rights and obligations of both parties shall be agreed upon separately during the guarantee period.',
    'title5':'5.※Remuneration and payment method:',
    'title5_description1': "Party A's payment of technical service payment and payment method to Party B is:",
    'title5_content1': '1.The total technical service fee is(￥):',
    'srvType': 'service type',
    'srvContent': 'service content',
    'amount': 'amount',
    'unitPrice': 'unit price',
    'money': 'money(￥)',
    'totalMoney': 'Total money',
    'title5_content2_1': '2.技术服务费由甲方按照以下第',
    'title5_content2_2': '种方式支付乙方。'

    
  },
  'canvas_canvas': {
    'clear': 'Clear', 'save': 'Save'
  },
  'my_my':{
    'myOrder': 'My Order', 'myDiscount': 'My Discount', 'myInfo': 'My Information', 'setting': 'Setting', 'changeLanguage': 'Language/语言', 'Language': ['English', '中文'], 'tabBar': { 'srvIntro': 'Service Intro', 'intelligentInquiry': 'Inquiry', 'ServicePurchase': 'Purchase', 'me': 'Me' },
  },
  'my_order':{
    'title': 'Order Information', 'order': 'Order', 'projectName': 'Project Name', 'sampleType': 'Sample Type', 'species': 'Species', 'taxonID': 'TaxonID', 'strainName': 'Strain Name', 'system': 'System', 'organ': 'Organ', 'status': 'Status', 'cellType': 'Cell Type', 'cell': 'Cell', 'fluid': 'Fluid', 'selectedService': 'Selected Services', 'analysisTime': 'Analysis Time', 'dataAnalysis': 'Data Analysis', 'sample':'Sample'
  },
  'my_discount':{
    'title': 'Promo Code', 'text': 'You can preview or long press to save into your album!', 'imgList': [
      { url: 'https://firmiana.ncpsb.org/static/we/discounts/0.1.png', count: '90% OFF' },
      { url: 'https://firmiana.ncpsb.org/static/we/discounts/0.2.png', count: '80% OFF' },
      { url: 'https://firmiana.ncpsb.org/static/we/discounts/0.3.png', count: '70% OFF' },
      { url: 'https://firmiana.ncpsb.org/static/we/discounts/0.4.png', count: '60% OFF' },
      { url: 'https://firmiana.ncpsb.org/static/we/discounts/0.5.png', count: '50% OFF' },
      { url: 'https://firmiana.ncpsb.org/static/we/discounts/0.6.png', count: '40% OFF' },
      { url: 'https://firmiana.ncpsb.org/static/we/discounts/0.7.png', count: '30% OFF' },
      { url: 'https://firmiana.ncpsb.org/static/we/discounts/0.8.png', count: '20% OFF' },
      { url: 'https://firmiana.ncpsb.org/static/we/discounts/0.9.png', count: '10% OFF' },
    ],
  },
  'userInfo_userInfo': {
    'title': 'User Information', 'name': 'Name', 'phoneNumber': 'Phone Number', 'mail': 'Mail', 'workUnit': 'Work Unit', 'department': 'Department', 'researchInterests': 'Research Interests'
  },
  'inquire_inquire': {
    'contact': 'Contact Customer Service'
  },
  'srvIntroduction_srvIntroduction': {
    'navigationBarTitle':'Service Introduction','srvProcess': 'Service Process', 'srvProduct': 'Service Product'
  },
  'srvIntroduction_srvProduct': {
    'title': 'Service Type', 'navigationBarTitle':'Service Introduction-Type Selection',
    'serviceTypeList': [
      {
        title: 'Qualitative Proteomics', id: 1, subtitle: [{ name: 'Proteome profiling', checked: false }, { name: 'Protein analysis after pull-down(IP,Co-IP)', checked: false }]
      },
      {
        title: 'Quantitative Proteomics', id: 2, subtitle: [{ name: 'Label-free quantification', checked: false }, { name: 'iTRAQ labelling quantification', checked: false }, { name: 'TMT labelling quantification', checked: false }, { name: 'SILAC labelling quantification', checked: false }]
      },
      {
        title: 'PTM Analysis', id: 3, subtitle: [{ name: 'Modified Proteomics analysis', checked: false }, { name: 'Phosphorylation Proteomics analysis', checked: false }, { name: 'Ubiquitylation  Proteomics analysis', checked: false }, { name: 'Glycosylation Proteomics analysis', checked: false }, { name: 'Acetylation Proteomics analysis', checked: false }]
      },
      {
        title: 'Transcription factor -DNA binding activity analysis', id: 4, subtitle: [{ name: 'Transcription factor -DNA binding activity analysis', checked: false }]
      }
    ]
  },
  'srvIntroduction_srvProcess': {
    'src': 'https://firmiana.ncpsb.org/static/we/workflow_E.png'
  },
  'staff_expresult_index_conclusions':{
    'myProject':'My Project','myInstrument':'My Instrument'
  },
  'staff_expresult_index_instrument': {
    'title': 'Instrument Information', 'instrument': 'Instrument', 'update': 'Update', 'status': 'Status', 'loading': 'Loading', 'warn': 'Warning', 'warn_content':'Request timed out, please try again later！','confirmText':'Confirm','cancelText':'Cancel'
  },
  'staff_expresult_index_instrumentDetail': {
    'chooseDate': 'Choose Date', 'reset': 'Reset', 'toTop': 'To Top', 'loading': 'Loading', 'warn': 'Warning', 'warn_content': 'The current WeChat version is too low. This function cannot be used temporarily. Please try again after upgrading.', 'confirmText': 'Confirm', 'cancelText': 'Cancel'
  },
  'staff_expresult_index_index':{
    'loading': 'Loading', 'projectName': 'Project Name', 'projectNumber': 'Project Number', 'organ': 'Organ', 'status': 'Status', 'species': 'Species', 'cell': 'Cell', 'experimentNumber': 'Experiment Number', 'progress': 'Progress', 'noMore': 'No more Projects', 'search':'search',
  },
  'staff_expresult_index_experiment': {
    'description': 'Description', 'status': 'Status', 'spectrum': 'Spectrum', 'peptides': 'Peptides', 'proteins': 'Proteins', 'genes': 'Genes', 'instrument': 'Instrument', 'loading': 'Loading', 'noMore': 'No more experiment', 'search': 'search','cancel':'cancel'
  },
  'srvIntroduction_srvIntroDetail': {
    'navigationBarTitle': '产品简介', 'list': [
      { theme: 'Product Definition', },
      { theme: 'Technical Principle', src: [] },
      { theme: 'Data Analysis', },
      { theme: 'Technical Advantages', },
      { theme: 'Application Field', },
      { theme: 'Sample Request', },
      { theme: 'References', }
    ],
    'srvIntroList': {
      'Qualitative Proteomics': [{
        "subType": 'Proteome profiling',
        'productDefinition': ['Long chromatographic gradient is currently a common method to achieve proteomic depth coverage, but the method is time consuming and the quantitative accuracy is not high. To achieve rapid depth coverage and accurate quantification of proteomics, the new technology were developed. It utilizes one-dimensional high-pH chromatography to fractionate complex protein mixtures and utilizes two-dimensional low-pH HPLC-MS/MS to achieve rapid depth coverage of proteomics.'],
        'technologyPrinciple': ['After protein extraction, reductive alkylation and Trypsin digestion, the peptides were fractionated by one-dimensional high pH chromatography, and the different fractions were subjected to two-dimensional low-pH HPLC-MS/MS. The entire process is optimized to achieve rapid depth coverage of proteomics within 12h.'],
        'dataAnalysis': ['1.Database identification data statistics', '2. Protein/peptide identification and quantitative results', '3. Protein/peptide identification and quantitative results', '4.GO analysis of protein and differential protein', '5.GO analysis of protein and differential protein', '6.GO analysis of protein and differential protein', '7.GO analysis of protein and differential protein'],
        'technicalAdvantages': ['1.Deep coverage of proteomics in a short period of time, covering up to 8000 proteins', '2.Deep coverage of proteomics in a short period of time, covering up to 8000 proteins', '3.Deep coverage of proteomics in a short period of time, covering up to 8000 proteins'],
        'applicationField': ['1.Deep coverage of proteomics in a short period of time, covering up to 8000 proteins', '2. Identification and quantification of Post-Modified Protein'],
        'requirements': [ 'Protein extract：≥50ug', 'Cell：≥10e5', 'Tissue：Animal Tissue ≥1mg,Plant Tissue≥10mg', 'Fluid：serum≥5μL,cerebrospinal fluid ≥10μL', 'If you have questions, please contact us.'],
        'reference': ['Ding C, Jiang J, Wei J, et al. A fast workflow for identification and quantification of proteomes. Molecular & Cellular Proteomics, 2013, 12(8): 2370-2380.'],
        'img_src': ['https://firmiana.ncpsb.org/static/we/imgTech/deepCover_figure.png']
      },
      {
        "subType": 'Protein analysis after pull-down(IP,Co-IP)',
        'productDefinition': ['With the analysis of proteins in purified samples such as IP, Co-IP, and Pull-down, the target protein, and the proteins that interact with the protein can be identified, so as to construct an interaction map.'],
        'technologyPrinciple': ['The target proteins obtained by IP, Co-IP, Pull-down，can be separated by SDS-PAGE and then subjected to in-gel digestion, or directly in solution, then the enzymatic products will be analyzed by high resolution mass spectrometry (LC-MS/MS).'],
        'dataAnalysis': ['1.Database identification data statistics', '2.Protein/peptide identification'],
        'technicalAdvantages': ['1. Identification of proteins suitable for IP, Co-IP, and Pull-down', '2. Identification of protein and protein interaction profiles with high sensitivity, high specificity, short experimental period and high throughput'],
        'applicationField': ['Identification of target protein and protein interaction spectrum'],
        'requirements': ['Purified protein: ≥10μg or protein bands visible in Coomassie blue staining gel'],
        'reference': [],
        'img_src': ['https://firmiana.ncpsb.org/static/we/imgTech/IP-CoIP_Process_E.png', 'https://firmiana.ncpsb.org/static/we/imgTech/IP-CoIP_figure.png']
      }],
      'Quantitative Proteomics': [{
        'subType': 'SILAC labelling quantification',
        'productDefinition': ['SILAC (Stable Isotope Labeling with Amino acids in Cell culture) 即Stable isotope labeling technology under cell culture conditions provides an effective solution for comprehensive and systematic quantitative analysis of proteomics. As a marker technology in vivo, SILAC hardly affects the function of cells, so it is currently widely used in proteomics-related fields.'],
        'technologyPrinciple': ['In cell culture, cell culture is carried out using essential amino acids containing light, medium and heavy isotopes (commonly used are stable isotope-labeled amino acids Lys and Arg). During cell subculture, isotopically labeled amino acids are used to synthesize proteins. After subculture for 5-6 generations, stable isotopically labeled amino acids are fully incorporated into newly synthesized proteins. After mixing the cells or the amount of protein of different labeled cells, the samples were separated, purified and subjected to mass spectrometry. The areas of the three labeled peptides in the first-order mass spectrum were compared for relative quantification, and the peptide fragments were sequenced using the secondary spectra.'],
        'dataAnalysis': ['1.Protein/peptide identification and quantitative results', '2.Protein/peptide identification and quantitative results', '3.Pathway metabolism analysis of protein and Pathway enrichment analysis of differential proteins', '4.Multi-sample classification statistics,functional enrichment,cluster analysis(for projects with sample number ≥ 3)'],
        'technicalAdvantages': ['1.High throughput for simultaneous identification and quantification of thousands of proteins', '2.Quantitative accuracy, wide linear range, and reduced experimental differences due to different sample preparation', '3.High compatibility, marking a variety of cells cultured in DMEM, DMEM-F12, 1640', '4.Marking efficiency is up to 100%, closer to the true state of the sample', '5.High sensitivity and low sample size'],
        'applicationField': ['Proteomic quantitative analysis for all types of cells that can be cultured'],
        'requirements': ['Protein extract：concentration >1ug/ul,total protein>300ug', 'Cell：≥10e7', ],
        'reference': ['1. Ong, S.E. & Mann, M. A practical recipe for stable isotope labeling by amino acids in cell culture (SILAC). Nat. Protoc. 1, 2650-2660 (2007).', '2. Sobczyk GJ, Wang J, et al. SILAC-based proteomic quantification of chemoattractant-induced cytoskeleton dynamics on a second to minute timescale. Nature Commun.2014; 5: 3319.', '3. Larance, M. et al. Stable-isotope labeling with amino acids in nematodes. Nature Methods. 8, 849-851 (2011).'],
        "img_src": ['https://firmiana.ncpsb.org/static/we/imgTech/SILAC_Process_E.png', 'https://firmiana.ncpsb.org/static/we/imgTech/SILAC_figure.png']
      },
      {
        'subType': 'Label-free quantification',
        'productDefinition': ['Label-free technology is a protein quantification technique that does not rely on isotope labeling. It directly analyzes protein enzymatic peptides by LC/MS. When analyzing mass spectrometry data, you can compare the mass spectrometric signal intensity and spectrum count of the same peptide in different samples to quantify the corresponding protein.'],
        'technologyPrinciple': ['Different samples are dealed with extracting and digesting proteins, grading and analyzing peptides respectively, the recorded data from each sample were compared for relative quantification of proteins. There are two ways to quantify proteins : one is based on peak intensity or peak area of ​ corresponding peptide of protein in MS1, and the other is based on spectral counts of corresponding peptide in MS2. '],
        'dataAnalysis': ['1.Database identification data statistics', '2.Protein/peptide identification and quantitative results', '3.Differential protein statistical analysis', '4.GO analysis of protein and differential protein', '5.Pathway metabolism analysis of protein and Pathway enrichment analysis of differential proteins', '6.Repetitive analysis (only for projects with repetitive design)', '7.Repetitive analysis (only for projects with repetitive design)'],
        'technicalAdvantages': ['1.No need to label and low cost', '2.Unrestricted by sample and samples in trace can be analyzed', '3.Wide dynamic detection-range to improve detection efficiency of low abundance proteins'],
        'applicationField': ['1.Identification and relative quantitation of protein in tissue or cell obtained from animal and plant, microorganisms, and body fluids', '2.Identification and relative quantitation of proteins enriched by pull-down or immunoprecipitation '],
        'requirements': ['Protein extract：≥50μg', 'Cell：≥10e6', 'Tissue：Animal Tissue ≥1mg,Plant Tissue≥10mg', 'Fluid：serum ≥5μL,cerebrospinal fluid ≥10μL', 'If you have questions, please contact us.'],
        'reference': ['1,Zhu W, J.W. Smith, and C.M. Huang. Mass spectrometry-based label-free quantitative proteomics. J BIOMED BIOTECHNOL. 2010; 2010: p. 840518.', '2,Ding C, Jiang J, Wei J, et al. A fast workflow for identification and quantification of proteomes, Mol Cell Proteomics. 2013 Aug;12(8):2370-80.', '3,Niemann M, Wiese S, et al. Mitochondrial outer membrane proteome of Trypanosomabrucei reveals novel factors required to maintain mitochondrial morphology. Mol Cell Proteomics. 2013; 12(2): 515-28.', '4,Cholewa BD, Pellitteri-Hahn MC, et al. Large-Scale Label-Free Comparative Proteomics Analysis of Polo-Like Kinase 1 Inhibition via the Small-Molecule Inhibitor BI 6727 (Volasertib) in BRAFV600E Mutant Melanoma Cells. J Proteome Res. 2014 Jun 9.', '5,Vanderschuren H, Nyaboga E, et al. Large-Scale Proteomics of the Cassava Storage Root and Identification of a Target Gene to Reduce Postharvest Deterioration. Plant Cell. 2014.'],
        "img_src": ['https://firmiana.ncpsb.org/static/we/imgTech/Label-Free_Process_E.png', 'https://firmiana.ncpsb.org/static/we/imgTech/Label-Free_figure.png']
      },
      {
        'subType': 'iTRAQ labelling quantification',
        'productDefinition': ['iTRAQ (Isobaric Tag for Relative Absolute Quantitation) and TMT (Tandem Mass Tags) technologies are in vitro labeling and quantification technologies for peptides developed by AB Sciex and Thermo, respectively, and are currently widely used in biology. The iTRAQ technology uses four or eight stable isotope tags. The TMT technology uses ten stable isotope tags that specifically label the amino group of the peptide and then perform tandem mass spectrometry to simultaneously compare the relative amounts of proteins in up to 10 different samples.'],
        'technologyPrinciple': ['The iTRAQ/TMT reagent consists of three parts: a reporter group, a balance group and a reactive group, which can form four, eight or ten equal mass isotope labels. These agents achieve efficient labeling of the enzymatically encoded polypeptide by binding to the amino terminus of the polypeptide and the primary amino group of the lysine residue. In the first-order mass spectrum, the same peptide from different samples showed the same mass-to-charge ratio after labeling. In tandem mass spectrometry, the bond between the reporter group, the equilibrium group and the reactive group, the same peptide with different isotopic labels produces reporter ions of different masses, and the same peptide between samples is obtained based on the signal intensity values of the reporter ions. The quantitative information is processed by software to obtain quantitative information of the protein.'],
        'dataAnalysis': ['1.Data identification and statistics ', '2.Protein/peptide identification and quantitative results', '3.Statistical analysis of differential protein ', '4.Pathway analysis of proteins and enrichment analysis of differential proteins', '5.Classification analysis,  function enrichment analysis, cluster analysis in multi-samples (applicable to samples with  number ≥ 3)'],
        'dataAnalysis': ['1.Database identification data statistics', '2.Protein/peptide identification and quantitative results', '3.Differential protein statistical analysis', '4.GO analysis of protein and differential protein', '5.Pathway metabolism analysis of protein and Pathway enrichment analysis of differential proteins', '6.Repetitive analysis (only for projects with repetitive design)', '7.Repetitive analysis (only for projects with repetitive design)'],
        'technicalAdvantages': ['1.Suitable for all various of biological samples', '2.Protein differential analysis can be carried out on 2-10 samples at the same time', '3.High protein flux and coverage'],
        'applicationField': ['Proteomic quantitative studies for different samples under various conditions.'],
        'requirements': ['Protein extract：≥100ug', 'Cell：≥10e7', 'Tissue：Animal Tissue ≥100mg,Plant Tissue≥10mg', 'Fluid：serum/plasma ≥500μL,urine≥25mL,saliva,cerebrospinal fluid ≥5ml', 'If you have questions, please contact us.'],
        'reference': ['1．Richard, D. U.et.al. Simultaneous analysis of relative protein expression levels across multiple samples using iTRAQ isobaric tags with 2D nano LC–MS/MS. Nature Protocols 5, 1574-1582(2010).', '2．Josselin Noirel1. Methods in Quantitative Proteomics: Setting iTRAQ on the Right Track. Current Proteomics 2011, 8, 17-30.', '3．Elias, J. E., Gygi, S. P. Target-decoy search strategy for increased confidence in large-scale protein identifications by mass spectrometry. Nat Methods 2007, 4(3):207–214.', '4．Altelaar A.F., et al. Benchmarking stable isotope labeling based quantitative proteomics. J Proteomics Oct 22. pii: S1874-3919(12)00704-X. doi: 10.1016/j.jprot.2012.10.009.', '5．Zhang, T., et al. (2010). Improving quantitation of TMT-labeled peptides using stepped higher-energy collisional dissociation. Application note # 483www.thermoscientific.com', '6．顾培明,李静. 应用Orbitrap Fusion对TMT标记样本进行MS2定量方法及SPS MS3定量方法的比较.塞默飞世尔（中国）有限公司.Application Notes CM0098.'],
        "img_src": ['https://firmiana.ncpsb.org/static/we/imgTech/iTRAQ-TMT_Process_E.png', 'https://firmiana.ncpsb.org/static/we/imgTech/iTRAQ-TMT_figure.png']
      },
      {
        'subType': 'TMT labelling quantification',
        'productDefinition': ['iTRAQ (Isobaric Tag for Relative Absolute Quantitation) and TMT (Tandem Mass Tags) technologies are in vitro labeling and quantification technologies for peptides developed by AB Sciex and Thermo, respectively, and are currently widely used in biology. The iTRAQ technology uses four or eight stable isotope tags. The TMT technology uses ten stable isotope tags that specifically label the amino group of the peptide and then perform tandem mass spectrometry to simultaneously compare the relative amounts of proteins in up to 10 different samples.'],
        'technologyPrinciple': ['The iTRAQ/TMT reagent consists of three parts: a reporter group, a balance group and a reactive group, which can form four, eight or ten equal mass isotope labels. These agents achieve efficient labeling of the enzymatically encoded polypeptide by binding to the amino terminus of the polypeptide and the primary amino group of the lysine residue. In the first-order mass spectrum, the same peptide from different samples showed the same mass-to-charge ratio after labeling. In tandem mass spectrometry, the bond between the reporter group, the equilibrium group and the reactive group, the same peptide with different isotopic labels produces reporter ions of different masses, and the same peptide between samples is obtained based on the signal intensity values of the reporter ions. The quantitative information is processed by software to obtain quantitative information of the protein.'],
        'dataAnalysis': ['1.Data statistics based on database', '2. Identification of protein/peptide and quantitative results', '3. Identification of protein/peptide and quantitative results', '4. Metabolic pathway analysis and enrichment analysis of differential proteins', '5. Classification analysis, function enrichment analysis, cluster analysis in multi-samples (applicable to samples with  number ≥ 3)'],
        'technicalAdvantages': ['1.Suitable for Various types of biological samples', '2.Suitable for Various types of biological samples', '3.1.	High protein throughput and coverage'],
        'applicationField': ['Can be used at proteomic quantitative studies for different samples under various conditions.'],
        'requirements': ['Protein extract：≥100ug', 'Cell：≥10e7', 'Tissue：Animal Tissue ≥100mg,Plant Tissue≥10mg', 'Fluid：serum/plasma ≥500μL,urine≥25mL,saliva,cerebrospinal fluid ≥5ml', 'If you have questions, please contact us.'],
        'reference': ['1．Richard, D. U.et.al. Simultaneous analysis of relative protein expression levels across multiple samples using iTRAQ isobaric tags with 2D nano LC–MS/MS. Nature Protocols 5, 1574-1582(2010).', '2．Josselin Noirel1. Methods in Quantitative Proteomics: Setting iTRAQ on the Right Track. Current Proteomics 2011, 8, 17-30.', '3．Elias, J. E., Gygi, S. P. Target-decoy search strategy for increased confidence in large-scale protein identifications by mass spectrometry. Nat Methods 2007, 4(3):207–214.', '4．Altelaar A.F., et al. Benchmarking stable isotope labeling based quantitative proteomics. J Proteomics Oct 22. pii: S1874-3919(12)00704-X. doi: 10.1016/j.jprot.2012.10.009.', '5．Zhang, T., et al. (2010). Improving quantitation of TMT-labeled peptides using stepped higher-energy collisional dissociation. Application note # 483www.thermoscientific.com', '6．顾培明,李静. 应用Orbitrap Fusion对TMT标记样本进行MS2定量方法及SPS MS3定量方法的比较.塞默飞世尔（中国）有限公司.Application Notes CM0098.'],
        "img_src": ['https://firmiana.ncpsb.org/static/we/imgTech/iTRAQ-TMT_Process_E.png', 'https://firmiana.ncpsb.org/static/we/imgTech/iTRAQ-TMT_figure.png']
      }
      ],
      "PTM Analysis": [
        {
          "subType": 'Modified Proteomics analysis',
          "productDefinition": ['Post-translational modification of proteins is closely related to many vital life processes, thus revealing that the law of post-translational modification is an important prerequisite for the analysis of complex and diverse biological functions of proteins. The purified protein was digested with Trypsin and detected by mass spectrometry. The data was analyzed and the post-translational modification parameters were set for database search to complete the post-translational modification.'],
          "technologyPrinciple": ['The target protein was obtained by IP, Co-IP, Pull-down, etc., and the enzymatic hydrolyzed product was subjected to high-resolution mass spectrometry (LC-MS/MS) after enzymatic hydrolysis or intra-solution hydrolysis. In the data analysis, post-translational modification parameters are set for post-translational modification identification. At present, various types of modification such as phosphorylation, glycosylation, acetylation and ubiquitination can be identified.'],
          "dataAnalysis": ['Identification of various modification types of proteins/peptides'],
          "technicalAdvantages": ['1.No need to enrich the modified protein, the identification coverage of the modified site is high', '2.Targeted detection of specified modification sites in combination with PRM technology', '3.Suitable for IP, Co-IP, Pull-down protein mixture sample identification', '4.High sensitivity, specificity, short experimental period'],
          "applicationField": ['Identification of post-translationally modified proteins in tissue, cells and body fluids under certain physiological or pathological conditions'],
          "requirements": ['Purified protein: ≥10μg or protein bands visible in Coomassie blue staining gel'],
          "reference": ['1. Aebersold R, Mann M. Mass spectrometry-based proteomics. Nature. 2003 Mar 13;422(6928):198-207.', '2. Witze ES1, Old WM, Resing KA, Ahn NG. Mapping protein post-translational modifications with mass spectrometry. Nat Methods. 2007 Oct;4(10):798-806.'],
          "img_src": ['https://firmiana.ncpsb.org/static/we/imgTech/Modification_Process_E.png', 'https://firmiana.ncpsb.org/static/we/imgTech/Modification_figure.png']
        },
        {
          "subType": 'Glycosylation Proteomics analysis',
          "productDefinition": ['Glycosylation modification is the process by which a protein is attached to a sugar under the control of an enzyme. N-glycosylation of proteins is one of the most important post-translational modifications of proteins, and plays a key role in complex multicellular or tissue formation processes.'],
          "technologyPrinciple": ['The N-glycosylation modification site of the protein has a conserved amino acid sequence NX(S/T), where X is an amino acid other than proline, and lectin recognizes and binds to a specific glycosyl sequence. Monosaccharide or glycan. The glycopeptide was captured by lectin and the sugar chain attached to the asparagine residue (Asn) was cleaved in H218O with N-glycosylase (PNGase F), resulting in an increase in Asn molecular weight of 2.9890 Da. The peptide fraction after de-sugarization was detected by high-precision mass spectrometry, and the change in molecular weight and its theoretical molecular weight after de-sugarization and the sequence of the glycosylation-modified peptide segment were confirmed to determine the N-glycosylation site. After protein extraction, trypsin digestion and desalting, the N-glycosylated peptide was enriched with lectin, and the sugar chain enriched in the glycopeptide was excised by PNGase F in H218O. Mass spectrometry then determines the glycosylation site. After the N-glycosylation site is determined, the label-free principle can be used for quantitative analysis.'],
          "dataAnalysis": ['1.Glycosylation modification site confirmation analysis', '2.Identification and quantification of intact glycopeptide structure', '3.GO analysis of glycoprotein and differential protein GO analysis', '4.Pathway Analysis of Glycoprotein and Pathway Analysis of Differential Proteins', '5.Repeatability analysis (only for projects with repetitive design)', '6.Multi-sample expression pattern cluster analysis (for projects with sample number ≥ 3)'],
          "technicalAdvantages": ['1.Lectin-specific enrichment of glycopeptides for a variety of sample types', '2.Quantitative analysis of glycosylation modification'],
          "applicationField": ['1.Changes in glycosylation-modified proteomics and signal pathway analysis under different conditions', '2.Screening for glycoprotein markers'],
          "requirements": ['Protein extract：≥2mg', 'Cell：≥10e7', 'Tissue：Animal Tissue ≥100mg,Plant Tissue≥100mg'],
          "reference": ['1.  Zielinska DF, Gnad F, et al. Mapping N-glycosylation sites across seven evolutionarily distant species reveals a divergent substrate proteome despite a common core machinery. Mol Cell. 2012; 46(4): 542-8.', '2. Qichen Cao, Cheng Ma, Haihong Bai, Xianyu Li, Hui Yan, Yan Zhao, Wantao Ying* and Xiaohong Qian*. Multivalent Hydrazide-Functionalized Magnetic Nanoparticles for Glycopeptide Enrichment and Identification. Analyst.2014, 139: 603-609'],
          "img_src": ['https://firmiana.ncpsb.org/static/we/imgTech/Glycosylation_Process_E.png', 'https://firmiana.ncpsb.org/static/we/imgTech/Glycosylation_figure.png']
        },
        {
          "subType": 'Phosphorylation Proteomics analysis',
          "productDefinition": ['Protein phosphorylation is the most basic, most common, and most important mechanism for regulating and controlling protein activity and function. It is closely related to many biological problems such as signal transduction, cell division, growth and development, and cancer mechanisms. Studying phosphorylation-modified proteomics is important for elucidating protein function.'],
          "technologyPrinciple": ['After the sample was extracted by protein, trypsin was hydrolyzed and desalted, the phosphorylated peptide was enriched with TiO2. The positively charged TiO2 is specifically bound to the negatively charged phosphorylated peptide under acidic conditions. Under the alkaline condition, the phosphorylated peptide was eluted from TiO2 to achieve the specific enrichment of the phosphorylated peptide. After the enrichment is completed, the enriched phosphorylated peptide is subjected to mass spectrometry. To maximize interference with negatively charged non-phosphorylated peptides in the peptide mixture, 2,5-dihydroxybenzoic acid or lactic acid is added to the TiO2 enrichment buffer. For quantitative analysis of phosphorylated proteins, iTRAQ labeling is performed prior to phosphorylation of peptides, and high-purity phosphorylated peptides are obtained by TiO2 enrichment. Finally, high-resolution mass spectrometry is performed to complete quantitative analysis.'],
          "dataAnalysis": ['1.Identification and quantification of phosphorylated peptides', '2.GO analysis of phosphorylated peptides and proteins and differential protein GO analysis', '3.Pathway analysis of phosphorylated peptides and proteins and pathway analysis of differential proteins', '4.Repeatability analysis (only for projects with repetitive design)', '5.Multi-sample expression pattern cluster analysis (for projects with sample number ≥ 3)'],
          "technicalAdvantages": ['1.Unique TiO2 enrichment buffer for specific enrichment of 95% phosphorylated peptides in samples', '2.Coverage of 8000-10000 phosphorylated peptides (about 3000-4000 proteins) in a short time', '3.Quantitative analysis'],
          "applicationField": ['1.Phosphorylation modified proteome changes and signal pathway analysis under different conditions', '2.Screening for phosphorylated modified protein substrates of specific protein kinases'],
          "requirements": ['Protein extract：≥2mg', 'Cell：≥10e7', 'Tissue：Animal Tissue≥100mg,Plant Tissue≥100 mg', 'Body Fluid：Serum ≥20μL，Cerebrospinal fluid≥30μL'],
          "reference": ['1. Thingholm TE, Jørgensen TJ, Jensen ON, Larsen MR. Highly selective enrichment of phosphorylated peptides using titanium dioxide. Nat Protoc. 2006;1(4):1929-35.', '2. Jensen SS, Larsen MR. Evaluation of the impact of some experimental procedures on different phosphopeptide enrichment techniques. Rapid Commun Mass Spectrom. 2007;21(22):3635-45.', '3. Nguyen TH, Brechenmacher L, et al. Quantitative phosphoproteomic analysis of soybean root hairs inoculated with Bradyrhizobium japonicum. Mol Cell Proteomics. 2012; 11(11): 1140-55.'],
          "img_src": ['https://firmiana.ncpsb.org/static/we/imgTech/Phosphorylation_Process_E.png', 'https://firmiana.ncpsb.org/static/we/imgTech/Phosphorylation_figure.png']
        },
        {
          "subType": 'Ubiquitylation  Proteomics analysis',
          "productDefinition": ['Ubiquitination refers to the process by which a ubiquitin molecule is specifically modified by a series of specific enzymes. Ubiquitination plays an important role in protein localization, metabolism, function, regulation and degradation, and is also involved in cell cycle, proliferation, apoptosis, differentiation, metastasis, gene expression, transcriptional regulation, signal transmission, injury repair, inflammatory immunity, etc. Almost all the regulation of life activities. In addition, ubiquitination is closely related to the onset of diseases such as cancer and cardiovascular disease.'],
          "technologyPrinciple": ['After digestion with Trypsin, the ubiquitinated lysine retains the diglycine tag, and the diglycine-modified peptide can be specifically enriched by the diglycine (diGG) antibody. After the sample was extracted by protein, trypsin was hydrolyzed and desalted, the obtained peptide was incubated with diGG antibody, and the diGG antibody was enriched and the ubiquitinated peptide was eluted, followed by desalting and mass spectrometry.'],
          "dataAnalysis": ['1.Identification and quantification of ubiquitinated peptides', '2.GO analysis of ubiquitinated peptides and proteins and differential protein GO analysis', '3.Pathway analysis of ubiquitinated peptides and proteins and pathway analysis of differential proteins', '4.Repeatability analysis (only for projects with repetitive design)', '5.Multi-sample expression pattern cluster analysis (for projects with sample number ≥ 3)'],
          "technicalAdvantages": ['1.High coverage of ubiquitinated peptides', '2.High-throughput ubiquitination identification'],
          "applicationField": ['1.Identification and quantification of ubiquitinated proteomics under different conditions', '2.Screening for specific E3 ubiquitin ligase substrates'],
          "requirements": [ 'Protein extract：≥5mg', 'Cell：≥10e8', 'Tissue：Animal Tissue ≥100mg,Plant Tissue≥100mg', 'Fluid：serum ≥20μL,cerebrospinal fluid ≥30μL'],
          "reference": ['GuoqiangXu, Jeremy S Paige, and Samie R Jaffrey. Global analysis of lysine ubiquitination by ubiquitin remnant immunoaffinity profiling. Nat Biotechnol. 2010 Aug; 28(8): 868–873.'],
          "img_src": ['https://firmiana.ncpsb.org/static/we/imgTech/Ubiquitination_Process_E.png', 'https://firmiana.ncpsb.org/static/we/imgTech/Ubiquitination_figure.png']
        },
        {
          "subType": 'Acetylation Proteomics analysis',
          "productDefinition": ["Acetylation refers to the process of adding an acetyl group to a lysine residue of a protein under the action of an acetyltransferase. This modification focuses on the effects on the chromosome structure of the cell and on the activation of transcriptional regulators in the nucleus. In recent years, the discovery of a large number of non-nuclear acetylated modified proteins reveals that acetylated modified proteins in life metabolism are much broader than people's knowledge and have a wide range of biological significance."],
          "technologyPrinciple": ['After the sample was extracted by protein, trypsin was hydrolyzed and desalted, the acetylated peptide was enriched with an acetylation-specific antibody. After the enrichment is completed, the enriched phosphorylated peptides are subjected to mass spectrometry, and the analytical data are retrieved by bioinformatics software.'],
          "dataAnalysis": ['1.Acetylation modification site confirmation analysis', '2.GO analysis of acetylated protein and differential protein GO analysis', '3.Pathway analysis of phosphorylated proteins and pathway analysis of differential proteins', '4.Repeatability analysis (only for projects with repetitive design)', '5.Multi-sample expression pattern cluster analysis (for projects with sample number ≥ 3)'],
          "technicalAdvantages": ['1.Acetylation-specific antibody-enriched peptides for a wide range of sample types', '2.Quantitative analysis of acetylation modification'],
          "applicationField": ['1.Analysis of acetylated modified proteome changes and signal pathway under different conditions', '2.Screening for acetylated modified protein markers'],
          "requirements": ['Cell：≥10e8', 'Tissue：Animal Tissue ≥100mg,Plant Tissue≥100mg', 'Fluid：serum/plasma ≥1mL'],
          "reference": ['1. Guo A, Gu H, et al. Immunoaffinity Enrichment and Mass Spectrometry Analysis of Protein Methylation. Mol Cell Proteomics. 2014; 13(1): 372-87.', '2. Tong Z, Kim MS, et al. Identification of Candidate Substrates for the Golgi Tul1 E3 Ligase Using Quantitative diGly Proteomic in Yeast. Mol Cell Proteomics. 2014; 13(11): 2871-82.', '3. Olsen JV, Blagoev B, et al. Global, in vivo, and site-specific phosphorylation dynamics in signaling networks. Cell. 2006; 127(3): 635-48.', '4. Cox J, Mann M. MaxQuant enables high peptide identification rates, individualized p.p.b.-range mass accuracies and proteome-wide protein quantification. Nat Biotechnol. 2008; 26(12): 1367-72.', '5. Lundby A, Secher A, et al. Quantitative maps of protein phosphorylation sites across 14 different rat organs and tissues. Nat Commun. 2012; 3: 876.'],
          "img_src": ['https://firmiana.ncpsb.org/static/we/imgTech/Acetylation_Process_E.png']
        }
      ],
      "Transcription factor -DNA binding activity analysis": [
        {
          "subType": 'Transcription factor -DNA binding activity analysis',
          "productDefinition": ['Due to the low content of transcription factors in cells and tissues, its determination at the proteomic level still faces enormous challenges. Transcription Factor DNA Binding Activity Profiling Technique Based on the binding of transcription factors to sequence-specific DNA elements, a tandem multicopy double-stranded DNA binding element (catTFRE) comprising 100 different types of transcription factor binding sequences was designed and synthesized. It is labeled with biotin to be packaged as a "DNA bait" to enrich endogenous transcription factors in cells or tissues for their identification at the proteomic level.'],
          "technologyPrinciple": ['The nuclear protein in the cell or tissue sample was extracted, and the obtained protein was subjected to SDS-PAGE separation, in-gel digestion and high-precision mass spectrometry by catTFRE-pull down technique, and transcription factor analysis was performed using professional software.'],
          "dataAnalysis": ['1.Data output statistics', '2.Transcription factor identification and quantitative results', '3.Downstream regulation of target genes in transcription factors'],
          "technicalAdvantages": ['1.Exclusive patented technology for deep coverage of endogenous transcription factors', '2.Combined with quantitative proteomics technology, it can simultaneously analyze the binding activity of hundreds or even thousands of transcription factors and DNA binding sequences.', '3.Dynamic monitoring of transcription factor expression and gene regulation'],
          "applicationField": ['Identification and quantification of large-scale transcription factors in animal cells or tissues under physiological conditions'],
          "requirements": ['Protein nuclear extract：≥500ug', 'Cell：≥10e7', 'Tissue：≥100mg'],
          "reference": ['Ding, Chen, et al. Proteome-wide profiling of activated transcription factors with a concatenated tandem array of transcription factor response elements. Proceedings of the National Academy of Sciences 110.17 (2013): 6771-6776.'],
          "img_src": ['https://firmiana.ncpsb.org/static/we/imgTech/TF-DNA_Process_E.png', 'https://firmiana.ncpsb.org/static/we/imgTech/TF-DNA_figure.png']
        }
      ]
    }
  },
  'simpleLearn_simpleLearnIndex': {
    'title': 'Learn About', 'List': [
      { name: 'Platform Introduction', src: './platformIntroduction', color: '#a0c8da' },
      { name: 'Service Product', src: '../srvIntroduction/srvProduct', color: '#b4c3ca' },
      { name: 'FAQ', src: './FAQ', color: '#c4bd6a' },
      { name: 'Service Process', src: '../srvIntroduction/srvProcess', color: '#d2b7a9' }
    ], 'register': 'register', 'text_head': 'Interested? Please', 'text_tail': 'and start shopping!',
  },
  'register_register':{
    'title': 'Account Registration for Research User', 'registered': 'You are registered!', 'name': 'Name', 'note': 'Please be sure to fill in your real name!', 'telephone': 'Telephone', 'mail': 'Mail', 'workUnit': 'Work Unit', 'department':'Department','name_PI':'PI name'
    , 'mail_PI': 'PI mail', 'researchInterests': 'Research Interests', 'password': 'Password', 'confirmPassword': 'Confirm Password', 'register':'register'
  },
  'simpleLearn_platformIntroduction':{
    'title': 'Fudan University iProteome Proteome Analysis Platform', 'subtitle1': '1.Basic Information', 'subtitle1_content': 'The iProteome proteome platform of Fudan University is a proteomics research platform jointly established by the School of Life Sciences, Phenotypic Research Institute and State Key Laboratory of Genetic Engineering of Fudan University. The iProteome proteomic analysis platform undertakes the teaching and research tasks of the school, provides technical support and talent training, and provides scientific research and technical services to the society. This platform provides proteomic qualitative, quantitative analysis, modified proteomic analysis and transcription factor DNA binding activity profiling. Includes one-stop proteomics sample preparation, proteomics testing, data analysis and more.\nThe platform technical support team is based on biology/medical doctors and masters, and the number of full-time technical support staff reaches 20.', 'subtitle2': '2.Platform Instrument', 'subtitle2_content': 'The platform currently includes two Orbitrap FusionTM LumosTM TribridTM 3-in-1 mass spectrometers with the ThermoFisher Q ExactiveTM HF-X combined quadrupole OrbitrapTM mass spectrometer.', 'subtitle3': '3.Contact Us', 'subtitle3_content':'Address: Fudan University Phenotypic Group Research Institute, 825 Zhangheng Road, Shanghai\n Tel:021-31240741'
  },
  'simpleLearn_FAQ': {
    'title': 'Commone Questions', 'QAList': [
      { Q: 'Q：Equipments？', A: 'A:The platform is equipped with multiple Q Exactive HF-X, Orbitrap Fusion Lumos and Orbitrap Fusion Lumos (with ETD) to meet the high quality and high standard identification requirements of each sample.' },
      { Q: 'Q：What can we do?', A: '①Proteomics qualitative analysis——Rapid depth coverage analysis of proteins\n②Proteomics quantitative analysis——SILAC labelling quantitation/ Label - Free quantitation / iTRAQ labelling quantitation / TMT labelling quantitation\n③Modified proteomic analysis——Qualitative analysis of protein modification sites / Glycosylation proteomics analysis / Phosphorylation Proteomics analysis / Ubiquitinated proteomics analysis / Acetylated proteomics analysis\n④Transcription factor - DNA binding activity profile analysis——Transcription factor - DNA binding activity profile analysis' },
      { Q: 'Q：Sample request？', A: 'A：The sample volume requirement can be provided according to the requirements of the “service product” corresponding project; samples of protein, cells and body fluids are stored in dry ice, and the peptides can be sent at room temperature.' },
      { Q: 'Q：Sample delivery process？', A: 'A：The user "registered account" - fill in the "sample information", submit the sample order - wait for the "order review" - prepare the barcode, paste it to the corresponding sample - mail sample, the specific operation can refer to "service flow".' },
      { Q: 'Q：Order Tracking？', A: 'A：The user can click on "My Order" and enter "Order Number" to view the "Order Current Process".' },
      { Q: 'Q：Complete time? (within 20)', A: 'A：The detection and feedback results were completed 3-4 weeks after the peptide sample was successfully received; the detection and feedback results were completed three months after the successful receipt of samples of protein, cells and body fluids; the accelerated samples were negotiated with the docking staff.' },
      { Q: 'Q：Data analysis type？', A: '1.Differential protein analysis\n2.Cluster analysis\n3.Pathway analysis' },
      { Q: 'Q：Sample price？', A: '' },
      { Q: 'Q：Project cooperation？', A: 'A：All people of insight are welcome to cooperate. You can contact us by email "Project Cooperation Public E-mail". We will have a dedicated staff to answer your questions, and you can also arrange related visits and negotiations.' },
    ]
  },
  'staff_metadata_metadata':{
    'title': 'Fudan University iProteome Analysis Platform', 'samplePreparation': 'Sample preparation', 'ms':'Mass spectrometry'
  },
  'staff_metadata_samplescan':{
    'title': 'Fudan University iProteme Platform', 'subtitle': 'Sample Preparation', 'placeholder':'Sample Number','scan':'Scan'
  },
  'staff_metadata_msscan':{
    'title': 'Fudan University iProteme Platform', 'subtitle': 'Mass spectrometry', 'placeholder': 'Sample Number', 'scan': 'Scan'
  }
 
  
}


module.exports = {
  content: English
}